src/style.css

```diff
@@ -1,3 +1,82 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+:root {
+  --active-collor: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header-nav {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  background-color: #fff;
+}
+
+.nav-bar {
+  margin: 0 50px 0 0;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav-list :first-child a {
+  margin: 0 10px 0 0;
+}
+
+.nav-list :last-child a {
+  margin: 0 0 0 10px;
+}
```

Comment:
if you need a margin in one direction - use `margin-right`, `margin-bottom` etc.